doc/components/extrema.md

```diff
@@ -0,0 +1,73 @@
+# Extrema
+
+ROHD-HCL provides a component to find the extrema of a list of Logic.
+
+The component `Extrema` will determine the extremas (maximum or minimum) and their position in a list of [`Logic`]s. 
+
+## Description
+
+`Extrema` will take in an input containing a `List<Logic>` along with parameters for `max` and `first`.
+
+* `max` is a boolean indicating whether to find the maximum value (`true`), or the minimum value (`false`). Default is `true`.
+
+* `first` is a boolean indicating whether to find the first instance of the extrema (`true`), or the last instance (`false`). Default is `true`.
+
+`Extrema` will then ouput two `Logic` signals:
+* `index` The index or position of the extrema in the list
+  >`index` will be a `Logic` with the smallest width needed to represent the largest possible index in the list.
+
+* `val` The value of the extrema.
+  >`val` will be a `Logic` with a width equal to the largest width of any element in the list.
+
+The `List<Logic>` may contain `Logic`s of any width. They will all be considered positive unsigned numbers. 
+
+`Extrema` will throw an exception if the `Logic` list is empty or has an empty element.
```

Comment:
what does an "empty element" mean?

Reply:
This was in reference to the test and exception thrown for a logic of some width but with no declared LogicValue.